url/urltestdata.json

```diff
@@ -6521,27 +6521,34 @@
     "search": "?a",
     "hash": "#%GH"
   },
-  "Bad bases",
+  "URLs that require a non-about:blank base. (Also serve as invalid base tests.)"
```

Comment:
This is missing a trailing comma.

Reply:
Thanks, I should actually test my changes against jsdom/whatwg-url as I do for everyone else's to this file -_-